secedgar/core/combo.py

```diff
@@ -0,0 +1,139 @@
+from datetime import date, timedelta
+
+from secedgar.core.daily import DailyFilings
+from secedgar.core.quarterly import QuarterlyFilings
+from secedgar.exceptions import EDGARQueryError
+from secedgar.utils import add_quarter, get_month, get_quarter
+
+
+class ComboFilings:
```

Comment:
@reteps how difficult would it be to make ComboFilings conform to AbstractFilings? I think that all we would need to do would be implement `get_urls` and add a few properties, but you probably know better than me.

Reply:
Added my thoughts to the main chain.